internal/config/main.go

```diff
@@ -119,16 +119,20 @@ func SetforTesting(key string, value string) error {
 	return k.Set(key, value)
 }
 func AdminUsernames() []types.Username {
-	usernames := []types.Username{}
-	for _, username := range k.Strings("admin_usernames") {
-		usernames = append(usernames, types.Username(username))
-	}
-	return usernames
+	return kUsernames("admin_usernames")
 }
 
 func TreOpsStaffUsernames() []types.Username {
+	return kUsernames("tre_ops_staff_usernames")
+}
+
+func IGOpsStaffUsernames() []types.Username {
+	return kUsernames("ig_ops_staff_usernames")
+}
```

Comment:
What does the `K` mean here?

Reply:
was trying to make it similar to `k.Strings` (where `k` is the config provider). better naming suggestions welcomed – maybe even just 

```go
func usernames()
```
?